app/src/layouts/common/appSidebar/helpAndService/FAQcontent/FAQContent.jsx

```diff
@@ -79,6 +81,7 @@ export const FAQContent = ({ onOpen, closeNavbar, closePopover }) => {
         id: 'requestSupportModal',
       }),
     );
+    trackEvent(HELP_AND_SUPPORT_EVENTS.CLICK_REQUEST_SUPPORT_BUTTON);
```

Comment:
`trackEvent` function should be used from [useTracking hook](https://www.npmjs.com/package/react-tracking#usage-with-react-hooks).
You can check, f.e. `addEditNotificationModal` component from `src/pages/inside/projectSettingsPageContainer/content/notifications/modals/addEditNotificationModal/addEditNotificationModal.jsx`